phoenix-tools/src/main/java/org/apache/phoenix/schema/SchemaSQLUtil.java

```diff
@@ -0,0 +1,151 @@
+package org.apache.phoenix.schema;
+
+import com.google.common.collect.ListMultimap;
+import org.apache.hadoop.hbase.util.Pair;
+import org.apache.phoenix.parse.ColumnDef;
+import org.apache.phoenix.parse.ColumnName;
+import org.apache.phoenix.parse.CreateIndexStatement;
+import org.apache.phoenix.parse.CreateTableStatement;
+
+import java.util.List;
+import java.util.Map;
+
+public class SchemaSQLUtil {
```

Comment:
Does this utility class belong in tools? (that is, is it only useful to a particular tool or tools?)

Reply:
I don't know where else it would be used at this time. Would be happy to move it else where as need arises.